ballerina-command/src/main/resources/cli-help/ballerina-dist-remove.help

```diff
@@ -0,0 +1,18 @@
+NAME
+      ballerina-remove - Remove distributions in your local environment
+
+SYNOPSIS
+      ballerina dist remove <distribution>
```

Comment:
This line is not aligned with line no 8 below.

Reply:
Ack